categories.xml

```diff
@@ -1,4 +1,5 @@
 <categories>
-	<category name="Uncategorized" slug="uncategorized"/>
+	<category name="Uncategorized" slug="uncategorized"></category>
+        <category name="Configuration and Utilities" slug="config"></category>
```

Comment:
Yeah, the identation I'll fix. Sorry about that.

`Configuration` and `Utilities` 'cause is the standart for others jQuery docs, similar case is qUnit docs.